packer/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+## Installs the Buildkite Agent, run from the CloudFormation template
+
+INSTANCE_ID=$(/opt/aws/bin/ec2-metadata --instance-id | cut -d " " -f 2)
+DOCKER_VERSION=$(docker --version | cut -f3 -d' ' | sed 's/,//')
+
+env
+
+# Cloudwatch logs needs a region specifically configured
+cat << EOF > /etc/awslogs/awscli.conf
+[plugins]
+cwlogs = cwlogs
+[default]
+region = $AWS_REGION
+EOF
+
+# cfn-env is sourced by the environment hook in builds
+cat << EOF > /var/lib/buildkite-agent/cfn-env
+BUILDKITE_STACK_NAME=$BUILDKITE_STACK_NAME
+BUILDKITE_SECRETS_BUCKET=$BUILDKITE_SECRETS_BUCKET
```

Comment:
We still need `$BUILDKITE_AGENTS_PER_INSTANCE` here for `/etc/terminationd/hook` to work. I've re-added it in 3935a9857b28bbecc2cd5cbb38cd58a315675a3f.

We noticed when our builds started seeing docker shut down mid-job, and found this in the logs:

```
Jan 12 23:37:00 ip-10-0-1-182 terminationd: Stopping buildkite-agent gracefully
Jan 12 23:37:00 ip-10-0-1-182 terminationd: /etc/terminationd/hook: line 12: BUILDKITE_AGENTS_PER_INSTANCE: unbound variable
Jan 12 23:37:00 ip-10-0-1-182 terminationd: Waiting for all buildkite-agent processes to have stopped...
```